src/shared/clients/codecatalystClient.ts

```diff
@@ -445,12 +492,17 @@ class CodeCatalystClientInternal {
 
     /**
      * Lists ALL of the given resource in the current account
+     *
+     * @param internalOnly If true, only internal repos will be returned.
      */
     public listResources(resourceType: 'org'): AsyncCollection<CodeCatalystOrg[]>
     public listResources(resourceType: 'project'): AsyncCollection<CodeCatalystProject[]>
-    public listResources(resourceType: 'repo'): AsyncCollection<CodeCatalystRepo[]>
+    public listResources(resourceType: 'repo', internalOnly?: boolean): AsyncCollection<CodeCatalystRepo[]>
```

Comment:
It's always `true`, do we need this param?

Reply:
When do we show all repos?

Reply:
Yeah, there are still places that want all repos. Plus its probably expected that you'd get all repos by default.